Document/Document.ipynb

```diff
@@ -13976,13 +14025,17 @@
     "\n",
     "<blockquote id=\"ref78\">78- G. W. Brier, \"Verification of forecasts expressed in terms of probability,\" <i>Monthly weather review</i>, vol. 78, no. 1, pp. 1-3, 1950.</blockquote>\n",
     "\n",
-    "<blockquote id=\"ref79\">79- L. Buitinck et al., \"API design for machine learning software: experiences from the scikit-learn project,\" <i>arXiv preprint arXiv:1309.0238</i>, 2013.</blockquote>\n"
+    "<blockquote id=\"ref79\">79- L. Buitinck et al., \"API design for machine learning software: experiences from the scikit-learn project,\" <i>arXiv preprint arXiv:1309.0238</i>, 2013.</blockquote>\n",
+    "\n",
+    "<blockquote id=\"ref80\">80- Hamming, Richard W. \"Error detecting and error correcting codes.\" The Bell system technical journal 29.2 (1950): 147-160.</blockquote>\n",
```

Comment:
@alirezazolanvari Would you please check the format of the references? (80, 81)
Then I will update `README.md`

Reply:
Hi @sepandhaghighi. Please replace them by the followings
"<blockquote id=\"ref80\">80- R. W. Hamming, \"Error detecting and error correcting codes,\" <i>The Bell system technical journal</i>, vol. 29, no. 2, pp. 147-160, 1950.</blockquote>\n",

"<blockquote id=\"ref81\">81- S. S. Choi, S. H. Cha, and C. C. Tappert, \"A survey of binary similarity and distance measures,\" <i>Journal of systemics, cybernetics and informatics</i>, vol. 8, no. 1, pp. 43-48, 2010.</blockquote>\n"

Reply:
Fix in https://github.com/sepandhaghighi/pycm/pull/443/commits/6787e41da5fde9d4582db76667709106b0e4e2f0